fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/properties/FluidProperty.java

```diff
@@ -0,0 +1,44 @@
+package net.fabricmc.fabric.api.fluids.v1.properties;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.nbt.Tag;
+
+/**
+ * a property of a fluid.
+ *
+ * @param <T> the type of data this property handles
+ */
+public interface FluidProperty<T extends Tag> {
+	FluidProperty<?> DEFAULT = new FluidProperty<Tag>() {
+		@Override
+		public boolean areCompatible(Fluid fluid, Tag aData, long aAmount, Tag bData, long bAmount) {
+			return true;
+		}
+
+		@Override
+		public Tag merge(Fluid fluid, Tag aData, long aAmount, Tag bData, long bAmount) {
+			return aData.copy();
+		}
+	};
+
+	/**
+	 * checks if 2 fluid volume's data are compatible with one another.
```

Comment:
`Checks if a FluidVolume's data is compatible with another's.`, maybe?

Either way, concerning grammar.

Reply:
yes I am proffessional english